test/support/exampleCall.js

```diff
@@ -0,0 +1,32 @@
+import jsonld from 'jsonld'
+import rdf from 'rdf-ext'
+import SparqlHttpClient from 'sparql-http-client'
+import frame from '../../src/frame.js'
+import { createApi } from '../../src/iiif.js'
+
+const client = new SparqlHttpClient({
+  endpointUrl: 'https://data.ptt-archiv.ch/query'
+})
+
+const clientOptions = {
+  operation: 'postUrlencoded'
+}
+
+const api = createApi(client, clientOptions)
+const url = 'https://data.ptt-archiv.ch/archive/instantiation/CH-000525-4%3Aaip-01-0000203993-0001/manifest'
+const iri = rdf.namedNode(url)
+
+const tryOut = async () => {
+  if (api.exists(iri)) {
+    const dataset = await api.getBasicDataset(iri)
+    const augmented = await api.augmentDataset(dataset)
+    const doc = await jsonld.fromRDF(augmented, {})
+    const framed = await frame(doc)
+    console.log(framed)
+  } else {
+    console.log('boo')
+  }
+}
+
+// @TODO make tests based on this
+tryOut()
```

Comment:
We should try to mock the endpoint answer, and then test to see if we get the expected results.
This will help a lot.

I will not require this for this PR, so that we can merge it right now, but this needs to be considered to be done soon.